src/openrct2/localisation/Localisation.h

```diff
@@ -112,11 +112,16 @@ class Formatter
         return CurrentBuf;
     }
 
-    void Increment(std::size_t count)
+    void Increment(size_t count)
     {
         CurrentBuf += count;
     }
 
+    void Rewind()
+    {
+        CurrentBuf -= NumBytes();
+    }
+
```

Comment:
Maybe?
```suggestion
    void Reset()
    {
        CurrentBuf = StartBuf;
    }

```

Reply:
Does not compile.

Reply:
It doesn't compile because the constness of StartBuf is incorrect. It should be a const pointer not a pointer to const data.  `uint8_t* const StartBuf;`. I leave it to a later PR to correct this.